go.mod

```diff
@@ -56,11 +56,11 @@ require (
 	google.golang.org/protobuf v1.34.2
 	gopkg.in/yaml.v3 v3.0.1
 	helm.sh/helm/v3 v3.14.4
-	istio.io/api v1.22.5
-	istio.io/client-go v1.22.0
-	k8s.io/api v0.29.9
+	istio.io/api v1.23.1
+	istio.io/client-go v1.23.1
+	k8s.io/api v0.30.0
```

Comment:
K8s dependencies will be updated by - https://github.com/gardener/gardener/pull/10459
/hold

Reply:
#10459 has been merged.

/unhold

@axel7born Pleas rebase.